integration/tests/cook/test_basic.py

```diff
@@ -59,7 +59,7 @@ def test_failing_submit(self):
 
     def test_progress_update_submit(self):
         command = 'echo "progress: 25 Twenty-five percent"; sleep 1; exit 0'
-        job_uuid, resp = util.submit_job(self.cook_url, command=command, max_runtime=5000)
+        job_uuid, resp = util.submit_job(self.cook_url, command=command, max_runtime=60000)
```

Comment:
Why did this need to change? 12x seems like a big bump

Reply:
On Travis, the test was failing due to "task max runtime exceeded"